torch/csrc/jit/pickle.h

```diff
@@ -58,10 +58,9 @@ TORCH_API void pickle(
 ///
 /// See `torch::pickle` for details.
 TORCH_API IValue unpickle(
```

Comment:
In general we should think a bit about how we want the Pickle C++ API to look at the end of this. @driazati, any thoughts? At least I think we should get rid of `tensor_table` if possible

Reply:
Any use of the `tensor_table` would be covered by pickle's [`persistent_id/load` functions](https://docs.python.org/2/library/pickle.html#pickling-and-unpickling-external-objects) (which we could implement if we need to in the future). This is already eager's serialization works anyways, so it's fine to get rid of the `tensor_table`